### PR TITLE
NODE: Add tools to generate documentation in README

### PR DESCRIPTION
**Pull request by daprahamian**
Ignore `README.md`, it is autogenerated from `etc/README.hbs`

### CODE REVIEW

File: bindings/node/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+MongoDB Client Encryption
+=========================
+
+The node.js wrapper for the Client Side Encryption library
```
Comment (mbroadst) [inline bindings/node/README.md]: `The Node.js wrapper for [libmongocrypt](..........)`

File: bindings/node/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+MongoDB Client Encryption
+=========================
+
+The node.js wrapper for [`libmongocrypt`](../../README.md)
```
Comment (kvwalker) [inline bindings/node/README.md]: Should node be capitalized?

File: bindings/node/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Encrypt a command for a given namespace
+
+<a name="AutoEncrypter+decrypt"></a>
+
+### *autoEncrypter*.decrypt(buffer, callback)
+
+| Param | Type |
+| --- | --- |
+| buffer | <code>\*</code> | 
+| callback | <code>\*</code> |
```
Comment (kvwalker) [inline bindings/node/README.md]: Does this need to be `<code>function</code>`?

File: bindings/node/README.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+Create a new encryption instance
+
+<a name="ClientEncryption+createDataKey"></a>
+
+### *clientEncryption*.createDataKey(provider, options, callback)
+
+| Param | Type | Description |
+| --- | --- | --- |
+| provider | <code>string</code> | The KMS provider used for this data key |
+| options | <code>\*</code> |  |
```
Comment (kvwalker) [inline bindings/node/README.md]: Should this be an object?

File: bindings/node/README.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+Creates a data key used for explicit encryption
+
+<a name="ClientEncryption+encrypt"></a>
+
+### *clientEncryption*.encrypt(value, options, callback)
+
+| Param | Type |
+| --- | --- |
+| value | <code>\*</code> | 
+| options | <code>\*</code> |
```
Comment (kvwalker) [inline bindings/node/README.md]: Same questions here.

File: bindings/node/README.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+Explicitly encrypt a provided value
+
+<a name="ClientEncryption+decrypt"></a>
+
+### *clientEncryption*.decrypt(value, callback)
+
+| Param | Type |
+| --- | --- |
+| value | <code>\*</code> | 
+| callback | <code>\*</code> |
```
Comment (kvwalker) [inline bindings/node/README.md]: And here.

File: bindings/node/README.md (hunk 3 above)
Comment (daprahamian) [reply bindings/node/README.md]: yes, but this is auto-generated. We have a separate issue for providing accurate API documentation [here](https://jira.mongodb.org/browse/NODE-2055).

**Pull request merged**